workflows4regtests/basic_examples/simple_workflow/parameters.json

```diff
@@ -44,17 +44,27 @@
         "file": "output/metaflow/AnnArbor_sub38614/save_json/segstats.json",
         "script": "check_output.py",
         "name": "AA38614"
+        },
+	{
+        "file": "output/metaflow/AnnArbor_sub46727/save_json/segstats.json",
+        "script": "check_output.py",
+        "name": "AA46727"
+        },
+        {
+        "file": "output/metaflow/AnnArbor_sub49687/save_json/segstats.json",
```

Comment:
i think there should be a test option that can operate on all the json files simultaneously instead of one at a time, and generate a dictionary that can be added automatically to the outer environment dictionary.

this is the data structures bit i was trying to convey but possibly failed miserably.

Reply:
@satra - you didn't fail to express it, but it was after I already created tests in a way that they take one file and compare with ref file. I know I should change it and make test more general, but just haven't done it yet.

Reply:
have to think what should the specification for test look like, if the one I choose (pretty simple) is not sufficient